src/parser.py

```diff
@@ -0,0 +1,64 @@
+import json
+import requests
+from datetime import date
+
+API_KEY = "NONE"
```

Comment:
We should not store the API key here because it will inenvitably lead us to commit them at some point and then we cannot open source the repo easily anymore

Reply:
Tips for how to do it e.g. here: https://stackoverflow.com/questions/44342276/how-to-push-code-to-github-hiding-the-api-keys